src/process_handling/breakpoint.rs

```diff
@@ -4,7 +4,7 @@ use nix::unistd::Pid;
 use nix::{Error, Result};
 use std::collections::HashMap;
 
-#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+#[cfg(any(target_arch = "x86", target_arch = "x86_64", target_arch = "aarch64"))]
```

Comment:
I was googling around to see what the equivalent instruction was in ARM to see if it's actually the same instruction and so far I've just got this on the LLDB mailing list: https://www.mail-archive.com/lldb-dev@lists.llvm.org/msg09058.html but it doesn't seem that they use the same opcode for software breakpoints...

Reply:
Yeah you'll want to find the bytes for the `BKPT` instruction. Also, if it's longer than a byte the `Breakpoint` type needs to change to handle varying length instructions depending on arch